dashboard/app/views/data_docs/show.html.haml

```diff
@@ -0,0 +1,2 @@
+%script{src: webpack_asset_path('js/data_docs/show.js'), data: {dataDocName: @data_doc_name.to_json, dataDocContent: @data_doc_content.to_json}}
```

Comment:
I see that this use of `to_json` works today, but is a little concerning because technically only things wrapped in `[]` and `{}` are valid json, and we have run into problems like this before when we try to upgrade. my recommendation would be to wrap these two fields in a single object like `data: {dataDocs: {...}.to_json}`. you could also just pass them as plain strings without to_json or getScriptData.

Reply:
Oo, good to know! I ran into an error without the `to_json` but didn't investigate further ... will look into changing this